sylvia-derive/src/message.rs

```diff
@@ -707,6 +707,29 @@ impl<'a> MsgVariants<'a> {
             }
         }
     }
+
+    pub fn emit_querier_for_bound_impl(
+        &self,
+        trait_module: &TokenStream,
+        contract_module: &TokenStream,
+    ) -> TokenStream {
+        let sylvia = crate_module();
+        let variants = &self.0;
+
+        let methods_impl = variants
+            .iter()
+            .filter(|variant| variant.msg_type == MsgType::Query)
+            .map(|variant| variant.emit_querier_impl(trait_module));
+
+        #[cfg(not(tarpaulin_include))]
+        {
+            quote! {
+                impl <'a, C: #sylvia ::cw_std::CustomQuery> #trait_module Querier for #contract_module BoundQuerier<'a, C> {
```

Comment:
Like here, see difference between `#sylvia ::cw_std::CustomQuery` (which obviously is the module where `cw_std`) and `#trait_module Querier` where it looks like it is some strange querier-prefix - not very obvious what to expect there to me.